SR2024/2024-03-15-competition-details.md

```diff
@@ -0,0 +1,63 @@
+---
+to: Student Robotics 2024
+subject: SR2024 Competition Details
+attachment: https://drive.google.com/file/d/1JQG_IKOtnvEUPSob1XWTma55FHRK_r4o/view
+---
+
+Hello everyone!
+
+It's just under a month until the big day - [the competition][competition-event-page]!
+
+Before then, we have a few important updates and reminders for you:
+
+## Attendance details
+
+Thanks to everyone who has already provided their attendance details. If you haven't done that yet, or things have changed substantially please [provide attendance details](https://forms.gle/8YhfdRbfDWP3KKNy8).
+
+Doors open on both days at 9am and each day will finish around 5:30pm. The event will formally start at 10:30am on Saturday. There are now full day schedules on the [event page][competition-event-page]. **If you are expecting to arrive after 10am on Saturday, please let us know.**
+
+You'll have access to your pits and the test arena when you arrive, giving your team time to work on their robot before things get going in earnest.
+
+The event is open to the public, so you're welcome to bring family and friends as guests, however please note that space in the pits is limited.
```

Comment:
Nit: Space is limited everywhere, not just the pits.

```suggestion
The event is open to the public, so you're welcome to bring family and friends as guests, however please note that space, particularly in the pits, is limited.
```

Reply:
Is this really the case? There's quite a lot of space in the Cube isn't there, which is where we're mainly expecting guests to go.

If we really do want to frame this as space being limited everywhere I think we should reduce the commas:
```suggestion
The event is open to the public, so you're welcome to bring family and friends as guests. However please note that space in the venue is limited, especially in the pits.
```

Reply:
I agree that wording is better.

There's space in the Cube, sure. But it's still worth saying "limited", just in case. Especially around the finals, even space in the cube gets to a premium.

And that's before we think of boring things like fire regs.

Reply:
57d063bd6fea58dcd769773117fa361a95cac5c0

Reply:
Arguably space _everywhere_ is limited -- the universe is finite after all -- but ok.